docs/business-apps/power-automate/get-started/connect-to-other-services-in-your-flow.md

```diff
@@ -1,67 +1,63 @@
-    * In the **My flows** tab, select **Send an email for new projects** flow.
-    * To edit the flow, select **Edit flow**.
+    * Select **Automate**, then select **Power Automate** and then expand to select **See your flows**.
+    * If prompted, sign in with your Microsoft 365 account.
+    * In the **My flows** tab, select **Send a customized email when a new SharePoint list item is added** flow.
+    * To edit the flow, in the flow details page, select **Edit** in the top command bar.
 
 3. In the flow designer, select **+ New step**.
 
-4. Select **Add an action**. The actions list opens where you can search and browse for various actions available in your flow in Power Automate.
+4. In the search box, to search for Teams-based actions, enter **Teams**.
 
-6. In the search box, to search for Teams-based actions, enter **Teams**.
+5. In the results, to filter the actions to just Teams, select the **Microsoft Teams** connector.
 
-7. In the results, to filter the actions to just Teams, select the **Microsoft Teams** connector.
+    ![Search for Microsoft Teams connector](../../../images/gs02-choose-action-microsoft-teams-connector.png)
 
-8. In the available actions list, select **Microsoft Teams - Post message**.
+6. In the available actions list, select **Microsoft Teams - Post message**.
     > To read more about the action and what it does, select the **information** icon.
 
     ![Add Team connector actions in your flow](../../../images/gs02-microsoft-teams-connector.png)
 
-9. As soon as the action is added, Power Automate begins to create the Teams connection. After the connection is created successfully, the action with the available inputs appears.
+7. As soon as the action is added, Power Automate begins to create the Teams connection. After the connection is created successfully, the action with the available inputs appears.
 
     ![Microsoft Teams post messsage action](../../../images/gs02-microsoft-teams-post-message-action.png)
 
-10. To see the connection information, select the **...** (ellipsis) button. To see the connection information the Teams action is using, check the **My connections** section.
+8. To see the connection information, select the **...** (ellipsis) button. To see the connection information the Teams action is using, check the **My connections** section.
 
-11. For the Teams action, enter the inputs.
-    * Team Id: From the picker, select a team.
-    * Channel Id: From the picker, select a channel.
+9. For the Teams action, enter the inputs.
+    * Team: From the picker, select a team.
+    * Channel: From the picker, select a channel.
 
-12. To construct a message that includes the project information, we must interact with the dynamic content panel. In the **Message**, enter the following in this same order:
+10. To construct a message that includes the project information, we must interact with the dynamic content panel. In the **Message**, enter the following in this same order:
```

Comment:
To construct a message that includes the project information, use the dynamic content panel.